src/test/script_tests.cpp

```diff
@@ -40,7 +40,7 @@ using namespace json_spirit;
 using namespace boost::algorithm;
 
 // Uncomment if you want to output updated JSON tests.
-// #define UPDATE_JSON_TESTS
+#define UPDATE_JSON_TESTS
```

Comment:
Is this supposed to stay uncommented in the actual commit, or is it supposed to be re-commented once the updated JSON has been created?